testapi.pm

```diff
@@ -390,8 +390,9 @@ sub upload_logs {
     my ($file) = @_;
 
     bmwqemu::log_call('upload_logs', file => $file);
-    my $cmd      = "curl --form upload=\@$file ";
     my $basename = basename($file);
+    my $upname   = ref($autotest::current_test) . '-' . $basename;
```

Comment:
IMO loadtest should fail - right now it stores the tests in %tests by $fullname. I would make that fatal by $name